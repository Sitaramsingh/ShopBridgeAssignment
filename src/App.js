
import 'bootstrap/dist/css/bootstrap.css';
import './App.css';
import Routes from './route';

function App() {
  return (
    <div className="App">
      <Routes/>
    </div>
  );
}

export default App;
