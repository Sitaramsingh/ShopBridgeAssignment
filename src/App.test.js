import React from 'react';
import { shallow } from 'enzyme';
import * as ReactRedux from 'react-redux'
import App from './App';


describe('Component should be render ', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<App/>);
    expect(component).toMatchSnapshot();
  });
    it('should work', () => {
      const mockXXXFn = jest.fn()
      const spyOnUseDispatch = jest
        .spyOn(ReactRedux, 'useDispatch')
        .mockReturnValue({ xxxFn: mockXXXFn })
      spyOnUseDispatch.mockRestore()
    })
});
