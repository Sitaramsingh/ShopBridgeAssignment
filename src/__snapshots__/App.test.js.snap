// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should be render  should render correctly in "debug" mode 1`] = `
<div
  className="App"
>
  <Routes />
</div>
`;
